README.md

```diff
@@ -21,10 +38,25 @@ Heya. I heard you like digging deep into Undertale data so I made a tool just fo
 * All core IO functionality extracted into a library for use in external tools.
 * Can generate an .yydebug file for the GM:S debugger so that you can edit variables live! (see [here](https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions#yydebug-file-format))
 
-### Included scripts
+# Screenshots
+
+Here's some screenshots of what UTMT can do:
+
+## [AM2R: Horde Mode Mod](https://github.com/Hornoads/AM2R-The-Horde-Multitroid/releases)
+<img src="images/am2r.png" alt="AM2R: Horde Mode" width="640" height="480"/>
```

Comment:
I got notified that the image here is actually just a mockup of a scrapped idea created in paint software.
Here's an actual image taken from the Mod
![image](https://user-images.githubusercontent.com/38186597/166960348-416dc464-77be-4ee9-a100-f2605a92774e.png)